setup.cfg

```
@@ -88,6 +88,7 @@ macos_platform =
     %(all_non_platform)s
 windows_platform =
     pywin32 != 226
+    twisted-iocpsupport ~= 0.0.1
```

Once there's support this distribution approach I'll do a v1 release

```suggestion
    twisted-iocpsupport ~= 1.0
```

---

yes. I think that 1.0 wil do. I guess this will allow for 1.0.1 or a future 1.0.2 bugfix release.